### PR TITLE
Set of improvements (column counts, automatic button class)

### PR DESCRIPTION
**Pull request by MartinMystikJonas**

### CODE REVIEW

**Comment by MartinMystikJonas** [comment]
@f3l1x Any idea why GitHUb actions did not run for this PR? I want to check if everything passes before I start make improvements. CUrrent build was failing.

**Comment by f3l1x** [comment]
CI was disabled due inactivity. You have to push once more. :-)

**Comment by MartinMystikJonas** [comment]
Waiting approval once again. Seem I was not approved permanently.

**Comment by f3l1x** [comment]
Can we merge it?

**Comment by MartinMystikJonas** [comment]
I could add neext commit with planned improvements. This one was just to set-up working CI before I start adding features.

**Comment by MartinMystikJonas** [comment]
Approval please ;-) Could you approve me permanently so I do not have to ask every time?

**Comment by MartinMystikJonas** [comment]
@f3l1x Could you review second commit tht added ability to set different columns widths? It it is on I will add third one with changed logic for btn-primary

**Comment by f3l1x** [comment]
So far so good. 👍

**Comment by MartinMystikJonas** [comment]
Added third commit with last change. Awaiting CI approval nad review. If it is ok you can merge it.

**Comment by MartinMystikJonas** [comment]
@f3l1x I kindly remind you I still need CI approved ;-)

**Comment by f3l1x** [comment]
Please reach me on Slack. Thx.

File: src/Rendering/AbstractBootstrapHorizontalRenderer.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<?php declare(strict_types = 1);
+
+namespace Contributte\Forms\Rendering;
+
+use Nette\Forms\Rendering\DefaultFormRenderer;
+
+class AbstractBootstrapHorizontalRenderer extends DefaultFormRenderer
```
Comment (f3l1x) [inline src/Rendering/AbstractBootstrapHorizontalRenderer.php:7]: abstract?

Comment (MartinMystikJonas) [reply src/Rendering/AbstractBootstrapHorizontalRenderer.php:7]: it is common ancestor of both v3 and v4 horizontal renderers with common functionality (setting column widths). Or should I just duplicate code in both renderers?

Comment (f3l1x) [reply src/Rendering/AbstractBootstrapHorizontalRenderer.php:7]: I ment `abstract class ..`.

Comment (MartinMystikJonas) [reply src/Rendering/AbstractBootstrapHorizontalRenderer.php:7]: Ah, yes I see. I forgot that. I will fix it right away.

**Pull request merged**